lib/pure/ioselects/ioselectors_poll.nim

```diff
@@ -208,7 +208,7 @@ proc newSelectEvent*(): SelectEvent =
   result.rfd = fds[0]
   result.wfd = fds[1]
 
-proc setEvent*(ev: SelectEvent) =
+proc trigger*(ev: SelectEvent) =
```

Comment:
This is a breaking change, please introduce a deprecated ``setEvent`` proc.

Reply:
https://github.com/nim-lang/Nim/pull/6585/files#diff-1d26d7d800ba43d0d051e86b6a1060d5R309